src/utils.cpp

```diff
@@ -27,4 +31,31 @@ namespace utils {
 
         return std::string{ buffer };
     }
+
+    [[nodiscard]] std::filesystem::path get_root_folder() {
+#if defined(__ANDROID__)
+        char* pref_path = SDL_GetPrefPath("coder2k", "oopetris");
+        if (!pref_path) {
+            throw std::runtime_error{ "Failed in getting the Pref Path on android!" };
+        }
+        return std::filesystem::path{ std::string{ pref_path } };
+#else
+        return std::filesystem::path{ "." };
+#endif
+    }
+
+    [[nodiscard]] std::filesystem::path get_assets_folder() {
+#if defined(__ANDROID__)
+        return std::filesystem::path{ "." };
+#else
+        return std::filesystem::path{ "assets" };
+#endif
+    }
+
+
+    [[nodiscard]] std::filesystem::path get_subfolder_to_root(std::string folder) {
```

Comment:
We should not take this parameter by value because this creates an unnecessary copy. Maybe we can use a `const std::string_view` here?